app/models/archimedes/function.rb

```diff
@@ -15,20 +17,20 @@ def all_functions
         ObjectSpace.each_object(::Class).select {|k| k < self }
       end
 
-      def is_func? function_name
-        self.name.split(/::/).last.snake_case == function_name.to_s
+      def is_func?(function_name)
+        self.name.split(/::/).last.underscore == function_name.to_s
```

Comment:
This is actually from Rails' ActiveSupport gem, which is also widely used outside of rails (e.g. I think bundler depends on it). String#snake_case and String#camel_case actually come from the gem 'extlib', which I use a lot just for this purpose.

Reply:
As long as it's not rails specific. I'll add it back in and update the gem files where 'extlib' was missing.